lib/policies_finder_publisher.rb

```diff
@@ -95,4 +98,22 @@ def publishing_api
     @publishing_api ||= PolicyPublisher.services(:publishing_api)
   end
 
+  def allowed_organisation_values
+    allowed_orgs = organiastions_used_by_policies.map do |org|
+      { label: org["title"], value: org["base_path"].split('/').last }
```

Comment:
I know we don't have much choice, but this looks quite brittle and highlight the disparity between content-store (&c) and search identifiers.

(not blocking, just noting)